creator-node/src/utils.js

```diff
@@ -74,13 +74,18 @@ async function getIPFSPeerId (ipfs, config) {
 
 const wait = (ms) => new Promise((resolve, reject) => setTimeout(() => reject(new Error('Timeout')), ms))
 
+/** Cat single byte of file at given filepath. */
 const ipfsSingleByteCat = (path, req) => new Promise(async (resolve, reject) => {
-  // Cat single byte
   const start = Date.now()
   let ipfs = req.app.get('ipfsAPI')
-  await ipfs.cat(path, { length: 1 })
-  req.logger.info(`ipfsSingleByteCat - Retrieved ${path} in ${Date.now() - start}ms`)
-  resolve('SUCCESS')
+  try {
```

Comment:
@SidSethi yes! we should maybe wrap all ipfs cat calls with this function beforehand. Not blocking this pr merge though